doc/source/calacs.rst

```diff
@@ -55,37 +55,26 @@ ACSSUM Python Wrapper
 CTE Forward Model
 =================
 
-This functionality is provided but not run as part of the pipeline.
+This functionality is provided to simulate the CTE effects associated with
+readout of the WFC detectors. It is not part of the standard CALACS pipeline.
 
 .. automodapi:: acstools.acscteforwardmodel
     :no-heading:
     :headings: =-
     :no-inheritance-diagram:
 
-PIXVALUE in FITS File
```

Comment:
Why is this removed?

Reply:
Well, I'm not sure this section reflects our actual recommendations to users. I use astropy.io.fits with acstools all the time and have never run into an issue, and we have example analysis notebooks for ACS data that do the same. Also, CALACS will produce an extension when all pixel values are the same, at least in some cases: https://github.com/spacetelescope/hstcal/issues/371#issuecomment-1031986684

Perhaps I should rewrite the section to recommend stsci.tools.stpyfits if users run into an issue?

Reply:
I think you will see this in RAW, where ERR and DQ are empty and represented by PIXVALUE. Or I think any calibrated data with homogeneous array, like if your DQ is somehow all zeroes.

Reply:
True. I put this section back in and updated it to hopefully make it more clear what may happen when using astropy with pipeline products, and recommend rather than require stpyfits.